appengine/mailgun/README.md

```diff
@@ -0,0 +1,62 @@
+## Mailgun on Google App Engine
+
+> [Mailgun](https://www.mailgun.com/): The Email Service For Developers
+
+### Sign up for Mailgun
+
+1. Sign up for a [Mailgun account](https://mailgun.com/signup).
+1. Add a [new domain](https://mailgun.com/app/domains).
+1. Find your API key in your new domain's settings.
+
+### Create a new Node.js app
+
+This example is going to use [Express.js](http://expressjs.com).
+
+### Configure
+
+Create an `app.yaml` in the root of your application with the following
+contents:
+
+```yaml
+runtime: nodejs
+vm: true
+api_version: 1
+env_variables:
+  PORT: 8080
+```
+
+Create a `config.js` in the root of your application with the following
```

Comment:
One of my major friction points on my initial project was: How do I provide config to my app in various environments? I couldn't set environment variables without committing config to my app's source control (not gonna do that), so I used a git-ignored config file.

If we want the little samples to be simpler by just committing config to source control, then for the samples that have config I think there should be a disclaimer that says something like "In a production app, here's how you might get config into your app: <link to article about it or link to built-for-production sample app, e.g. LabelCat>"